docs/cmdline-opts/quote.d

```diff
@@ -26,6 +26,9 @@ itself before sending them to the server.  File names may be quoted
 shell-style to embed spaces or special characters.  Following is the list of
 all supported SFTP quote commands:
 .RS
+.IP "atime 'unix timestamp' file"
+The atime command sets the last access time of the file named by the file
+operand. The time is expressed as elapsed seconds since Jan 1, 1970 (UTC).
```

Comment:
Elapsed seconds since Jan 1, 1970 is a very "hostile" way of specifying a time or date. Couldn't we perhaps use the time/date parser we already have and allow more human readable strings here? Using `Curl_getdate_capped()` perhaps?

Reply:
My OS handles it that way: `Aug  5 09:14`, why not use this?